release-1.8/release_notes_draft.md

```diff
@@ -37,19 +37,21 @@ please check out the [release notes guidance][] issue.
 
 ## **Major Themes**
 
-- The kubernetes workloads API (the DaemonSet, Deployment, ReplicaSet, and
-StatefulSet kinds) have been moved to the new apps/v1beta2 group version. This
-is the current version of the API, and the version we intend to promote to
-GA in future releases. This version of the API introduces several deprecations
-and behavioral changes, but its intention is to provide a stable, consistent
-API surface for promotion.
-
 - The roles based access control (RBAC) API group for managing API authorization
 has been promoted to v1. No changes were made to the API from v1beta1. This
 promotion indicates RBAC's production readiness and adoption. Today, the
 authorizer is turned on by default by many distributions of Kubernetes, and is a
 fundamental aspect of a secure cluster.
 
+### SIG Apps
+
+[SIG Apps][] is responsible for deploying and operating applications in Kubernetes, and focuses on the developer and devops experience of running applications.
```

Comment:
deploying, operating, and running may be a little too verb heavy? I see that this is basically what SIG apps has on their community page but perhaps that could use some wordsmithing as well

Reply:
as discussed offline, will use `SIG Apps focuses on the Kubernetes APIs and external tools required to support the deployment and operation of a wide variety of workloads.`